scripts/update-apk.sh

```diff
@@ -60,18 +60,14 @@ for file in app*; do
 done
 
 if $IS_PUBLISH_BRANCH ;then
-    # Move to previous directory so that fastlane metadata gets uploaded
-    cd ..
     gem install fastlane
-    fastlane supply --aab ./apk/eventyay-attendee-master-app-playStore-release.aab --skip_upload_apk true --track alpha --json_key ./scripts/fastlane.json --package_name $PACKAGE_NAME $FASTLANE_DRY_RUN
+    fastlane supply --aab eventyay-attendee-master-app-playStore-release.aab --metadata_path ../fastlane/metadata/android --skip_upload_apk true --track alpha --json_key ../scripts/fastlane.json --package_name $PACKAGE_NAME $FASTLANE_DRY_RUN
```

Comment:
Shouldn't it be `fastlane/metadata`?

Reply:
@iamareebjamal no, after testing it with my personal application I found that it looks for the folder which has en-US folder as its direct child.

Reply:
Here is a screenshot of a successful deployment. 
<img width="640" alt="Screen Shot 2019-12-10 at 10 10 15 PM" src="https://user-images.githubusercontent.com/11988517/70573461-f013d800-1b99-11ea-8f02-cf376e752d5a.png">